test/test/HelloFunctionalTest.java

```diff
@@ -1,7 +1,7 @@
 package test;
 
 import akka.grpc.GrpcClientSettings;
-import akka.grpc.play.JavaAkkaGrpcClientHelpers;
```

Comment:
not part of this PR but I find using `test` as package name confusing.
(note how the file is on the `test/test/` folder)

Use `com.example` instead?

Reply:
Yeah, we can move them.

Reply:
raised https://github.com/playframework/akka-grpc-play-quickstart-java/issues/22